src/Compilers/Core/Portable/Syntax/SyntaxToken.cs

```diff
@@ -13,12 +13,9 @@
 
 namespace Microsoft.CodeAnalysis
 {
-#pragma warning disable CA1200 // Avoid using cref tags with a prefix
     /// <summary>
-    /// Represents a token in the syntax tree. This is the language agnostic equivalent of <see
-    /// cref="T:Microsoft.CodeAnalysis.CSharp.SyntaxToken"/> and <see cref="T:Microsoft.CodeAnalysis.VisualBasic.SyntaxToken"/>.
```

Comment:
this seems reasonable to me.  Looks like this comment was before we unified on the shared-syntax model.